plugins/qlcplus_4.12.1/import.js

```diff
@@ -415,6 +432,177 @@ const qlcPlusChannelPresets = {
   })
 };
 
+const getShutterStrobeCap = (shutterEffect, speedStart, speedEnd, randomTiming) => {
+  const cap = {
+    type: `ShutterStrobe`,
+    shutterEffect
+  };
+
+  if (speedEnd) {
+    cap.speedStart = speedStart;
+    cap.speedEnd = speedEnd;
+  }
+  else if (speedStart) {
+    cap.speed = speedStart;
+  }
+
+  if (randomTiming) {
+    cap.randomTiming = true;
+  }
+
+  return cap;
+};
+
+const getRotationCapType = ({ channelName, channelType }) => {
+  if ([`Pan`, `Tilt`].includes(channelType)) {
+    return `${channelType}Continuous`;
+  }
+
+  if ([`Colour`, `Gobo`].includes(channelType) || /\bgobo\b/i.test(channelName)) {
+    return /wheel\b/i.test(channelName) ? `WheelRotation` : `WheelSlotRotation`;
+  }
+
+  if (channelType === `Prism`) {
+    return `PrismRotation`;
+  }
+
+  return `Rotation`;
+};
+
+const getRotationSpeedCap = (capData, speedStart, speedEnd) => {
+  const cap = {
+    type: getRotationCapType(capData)
+  };
+
+  if (speedEnd) {
+    cap.speedStart = speedStart;
+    cap.speedEnd = speedEnd;
+  }
+  else if (speedStart) {
+    cap.speed = speedStart;
+  }
+
+  return cap;
+};
+
+const qlcPlusCapabilityPresets = {
+  SlowToFast: () => ({
+    type: `Speed`,
+    speedStart: `slow`,
+    speedEnd: `fast`
+  }),
+  FastToSlow: () => ({
+    type: `Speed`,
+    speedStart: `fast`,
+    speedEnd: `slow`
+  }),
+  NearToFar: () => ({
+    type: `Focus`,
+    distanceStart: `near`,
+    distanceEnd: `far`
+  }),
+  FarToNear: () => ({
+    type: `Focus`,
+    distanceStart: `far`,
+    distanceEnd: `near`
+  }),
+  BigToSmall: () => ({
+    type: `Iris`,
+    openPercentStart: `open`,
+    openPercentEnd: `closed`
+  }),
+  SmallToBig: () => ({
+    type: `Iris`,
+    openPercentStart: `closed`,
+    openPercentEnd: `open`
+  }),
+  ShutterOpen: () => ({
+    type: `ShutterStrobe`,
+    shutterEffect: `Open`
+  }),
+  ShutterClose: () => ({
+    type: `ShutterStrobe`,
+    shutterEffect: `Closed`
+  }),
+  StrobeSlowToFast: () => getShutterStrobeCap(`Strobe`, `slow`, `fast`),
+  StrobeFastToSlow: () => getShutterStrobeCap(`Strobe`, `fast`, `slow`),
+  StrobeRandom: () => getShutterStrobeCap(`Strobe`, null, null, true),
+  StrobeRandomSlowToFast: () => getShutterStrobeCap(`Strobe`, `slow`, `fast`, true),
+  StrobeRandomFastToSlow: () => getShutterStrobeCap(`Strobe`, `slow`, `fast`, true),
```

Comment:
Should be `` getShutterStrobeCap(`Strobe`, `fast`, `slow`, true), ``

Reply:
Fixed in [`ba62e01`](https://github.com/OpenLightingProject/open-fixture-library/pull/978/commits/ba62e01e49e7515b11a4d924e7b7730bbe876cc4#diff-83e5c40e8119c68f3a2416ac07053e96R699)